backend/services/automations/src/executions/checkIsWaitingActionTarget.ts

```
@@ -0,0 +1,85 @@
+  const automation = await models.Automations.findOne({
+    _id: waitingAction.automationId,
+  });
+
+  for (const action of automation?.actions || []) {
+    const connects = action.config?.optionalConnects || [];
+
+    const optionalConnect = connects.find(
+      ({ optionalConnectId }) => optionalConnectId === valueToCheck,
+    );
+
+    if (optionalConnect && optionalConnect?.actionId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)